pom.xml

```diff
@@ -195,11 +193,6 @@
 				<artifactId>spring-cloud-deployer-cloudfoundry</artifactId>
 				<version>${spring-cloud-deployer-cloudfoundry.version}</version>
 			</dependency>
-			<dependency>
```

Comment:
We don't get dependency from parent, we just get dependency management. We still need to define it but without version.

Reply:
Yep, this was doing `<dependencyManagement>` for spring-shell in SCDF. Now that we have lifted up higher to SCDB we can remove it here. The actual dependency is still specified in [spring-cloud-dataflow-shell-core/pom.xml](https://github.com/spring-cloud/spring-cloud-dataflow/blob/a66193047b72e76162bcc2422cf4bbd00aa0858d/spring-cloud-dataflow-shell-core/pom.xml#L22).

Reply:
Ah stupid me, I was looking wrong file.